src/status_im/multiaccounts/key_storage/core.cljs

```diff
@@ -264,6 +266,16 @@ We don't need to take the exact steps, just set the required state and redirect
             {:db (assoc db :goto-key-storage? true)}
             (multiaccounts.logout/logout)))
 
+(rf/defn open-device-settings-dialog
+  {:events [::open-device-settings-dialog]}
```

Comment:
we are trying to avoid using namespaced keywords

Reply:
Perhaps `:multiaccounts/open-device-settings-dialog` ? Or do you have some alternative suggestions @flexsurfer ?

Reply:
i think the namespace is wrong, im not sure what the device settings dialog is

Reply:
Need to move this code to somewhere else like scan/events.cljs

Reply:
yes please

Reply:
updated